### DISCUSSION TITLE
87

### DISCUSSION THREAD

**Discussion by giscus** [sukvvon/blog-comments#25] [Comments]
# 87

새로운 시작 말로만 듣고 온라인으로 github repository를 참고하기만 했던 프론트엔드 데브코스가 시작되어 버렸다... 지금 이 글을 쓰고 있는 지금도 내가 이 과정에 참여하고 있다는 것이 믿겨지지가 않는다. 만남 3기 tooooo1님과의 만남 약 1년 전부터 github 상기로 혼자 알고만 있던 tooooo1님과 프론트엔드 데브코스에서 운영중인 디스코드에서 이야기를 할 수 있는 기회를 가지게 되었다. 마치 연예인을 보는 듯한 기분이랄까...? 이야기를 하다보니 같은 동네에 살게 된다는 것을 알게되었고, 시간은 새벽 1시였지만 만나게 되었다. 그동안 github상으로 궁금한 코드들도 많았고, 평상시에 어떻게 생각하시는지, 막힐 때 어떻게 대처하는지, 어떻게 삽질(?)을 하시는지 물어보고 직접 하..

https://sukvvon.tistory.com/87

**Comment by MinwooP** [comment]
회고글 잘 읽었습니다 ! 7월도 화이팅입니다 원석님 🤗
